generators/entity-client/templates/ionic/e2e/entities/_entity.e2e-spec.ts.ejs

```diff
@@ -0,0 +1,205 @@
+<%#
+ Copyright 2013-2019 the original author or authors from the JHipster project.
```

Comment:
This should be 2020-Present since you just created it. Or 2019-Present to be consistent with the rest of the templates.